### ISSUE TITLE
Library not able to automatically update 

### ISSUE THREAD

**Issue by jkl1234567** [inorichi/tachiyomi#1428]
**App version:**
0.6.6-0.7.4
**Issue/Request:**
Automatic update not working for library. Library possibly also unable to manually update.  
**Steps to reproduce (if applicable)**

 1. Set library to update every 6 hours with no restrictions
 2. Notifications are turned off for the tachiyomi app.
 3. Library does not update automatically and manual library does not appear to update the library. 

**Other details:**
Normally when notifications are on for the tachiyomi app, there is a notification while tachiyomi app is updating. However, after 2-3 days since installing the tachiyomi app, turning notifications back on for the tachiyomi app does not produce a updating notification when attempting to manually updating the library, leading to the conclusion that the library possibly is not able to update at all. I have tried different versions in the last 3 months and occasionally I am able to get the library automatically updating, however after a period of use, the library would cease to update.

**Comment by jkl1234567** [comment]
After a bit of research I have found that occasionally when the library is updating the update would get stuck on a chapter and would stay that way. This prevents the automatic update from triggering as well as the manual update as the library is still 'updating'. As I had notifications turned off previously I had no way to cancel the update and the library was unable to update. Currently I have left notifications on to monitor this issue. I have browsed the forums and there was a similar issue back in 2016 for an older version but I have noticed this issue amongst the versions that I have been using so the issue or a similar form of the issue still persists in these versions.
